serverless-test-project-alb/sls-resources.yml

```diff
@@ -0,0 +1,125 @@
+Resources:
+  bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: awesome-savage-bucket1
+
+
+  vpcALB:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.0.0.0/20
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      InstanceTenancy: default
+      Tags: ${self:custom.resourceTags}
+
+  internetGateway:
+    Type: AWS::EC2::InternetGateway
+    Properties:
+      Tags: ${self:custom.resourceTags}
+  
+  vpcGatewayAttachment:
+      Type: AWS::EC2::VPCGatewayAttachment
+      Properties:
+        VpcId: !Ref vpcALB
+        InternetGatewayId: !Ref internetGateway
+
+  subnetA:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: ${self:provider.region}a
+      CidrBlock: 10.0.5.0/24
+      VpcId: !Ref vpcALB
+      Tags: ${self:custom.resourceTags}
+
+  subnetB:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: ${self:provider.region}b
+      CidrBlock: 10.0.6.0/24
+      VpcId: !Ref vpcALB
+      Tags: ${self:custom.resourceTags}
+  
+  routeTableA: 
+      Type: AWS::EC2::RouteTable
+      Properties:
+        VpcId: !Ref vpcALB
+        Tags: ${self:custom.resourceTags}
+  
+  routeTableB: 
+      Type: AWS::EC2::RouteTable
+      Properties:
+        VpcId: !Ref vpcALB
+        Tags: ${self:custom.resourceTags}
+
+  routeTableAssociationSubnetA:
+      Type: AWS::EC2::SubnetRouteTableAssociation
+      Properties:
+        SubnetId: !Ref subnetA
+        RouteTableId: !Ref routeTableA
+  
+  routeTableAssociationSubnetB:
+      Type: AWS::EC2::SubnetRouteTableAssociation
+      Properties:
+        SubnetId: !Ref subnetB
+        RouteTableId: !Ref routeTableB
+
+  routeTableAInternetRoute:
+      Type: AWS::EC2::Route
+      DependsOn: vpcGatewayAttachment
+      Properties:
+        RouteTableId: !Ref routeTableA
+        DestinationCidrBlock: 0.0.0.0/0
+        GatewayId: !Ref internetGateway
+
+  routeTableBInternetRoute:
+      Type: AWS::EC2::Route
+      DependsOn: vpcGatewayAttachment
+      Properties:
+        RouteTableId: !Ref routeTableB
+        DestinationCidrBlock: 0.0.0.0/0
+        GatewayId: !Ref internetGateway
+
+  alb:
```

Comment:
If you add a second load balancer to the project, it should reveal the issue with looking up the load balancer for a target group.